material-stepper/src/main/res/layout/ms_step_tab.xml

```diff
@@ -27,6 +27,15 @@
             app:srcCompat="@drawable/ic_check"
             android:visibility="gone" />
 
+        <android.support.v7.widget.AppCompatImageView
```

Comment:
Is a separate ImageView needed here? Maybe we could use ms_stepDoneIndicator and change the drawable + tint? What do you think?